src/hooks/init/plugin-install.js

```diff
@@ -0,0 +1,35 @@
+const prompt = require('inquirer').prompt;
+const { logger } = require('@twilio/cli-core').services.logging;
+const ALLOWED_ORGS = [
+  '@twilio',
+  '@twilio-labs'
```

Comment:
Should use trailing `/` so someone doesn't just make an `@twilio-schmilio` org that passes.